pkg/apis/acid.zalan.do/v1/util.go

```diff
@@ -102,5 +102,5 @@ func (postgresStatus PostgresStatus) Creating() bool {
 }
 
 func (postgresStatus PostgresStatus) String() string {
-	return fmt.Sprintf(`"status":{"PostgresClusterStatus": %s}`, postgresStatus.PostgresClusterStatus)
+	return fmt.Sprintf(`{"status":{"PostgresClusterStatus": %s}}`, string(postgresStatus.PostgresClusterStatus))
```

Comment:
Can we assume that string() does the proper quote/escape?

Reply:
After short discussion we elaborate on going back to marshal as the proper solution to generate json of nested objects.